web/app/components/workflow/nodes/_base/panel.tsx

```diff
@@ -150,6 +151,7 @@ const BasePanel: FC<BasePanelProps> = ({
                   </Tooltip>
                 )
               }
+              <NodePosition nodeId={id}></NodePosition>
```

Comment:
Suggestion:
Use data directly in `NodePosition `, so that you don't need to get the node by `const node = reactflow.getNode(nodeId)`